src/click/testing.py

```diff
@@ -116,6 +116,13 @@ def name(self) -> str:
     def mode(self) -> str:
         return self._mode
 
+    def __next__(self) -> str:  # type: ignore
```

Comment:
mypy claims the return type should be `bytes` because of `IOBase` but it should be matching with `TextIOWrapper`